src/genegraph/database/query.clj

```diff
@@ -98,8 +98,9 @@
   "Create a new model that is the union of models"
   [& models]
   (let [union-model (ModelFactory/createDefaultModel)]
-    (doseq [model models] (.add union-model model))
-    union-model))
+    (util/tx
+     (doseq [model models] (.add union-model model))
```

Comment:
Is it necessary to do this in a transaction? Does this interact with the database?

Reply:
Removed,